config/initializers/devise.rb

```diff
@@ -224,7 +224,7 @@
   # config.navigational_formats = ['*/*', :html]
 
   # The default HTTP method used to sign out a resource. Default is :delete.
-  config.sign_out_via = :delete
+  config.sign_out_via = :get
```

Comment:
Why this change? Ideally, `GET` requests should be idempotent, while `DELETE` seems very appropriate here.